src/crawlers/cheerio_crawler.js

```diff
@@ -40,6 +40,9 @@ const DEFAULT_OPTIONS = {
         },
     },
     additionalMimeTypes: [],
+    useSessionPool: false,
+    sessionPoolOptions: {},
+    persistCookiesPerSession: false,
```

Comment:
Why is this not an option of the SessionPool?

Reply:
Because it is not related to SessionPool, Sessionpool cannot persist cookies, because it has no access to the request object it should serve only as storage. Let's leave the cookies logic on the Crawlers since it is handled differently in the cheerio and puppeteer.

Reply:
So lets put cookie jar for each session and let the crawlers use its interface.